cmd/validate/image.go

```diff
@@ -140,6 +142,23 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 				data.spec = s
 			}
 
+			// Check if policyConfiguration is a file path, if so, we read it into the var data.policyConfiguration
+			if strings.Contains(data.policyConfiguration, ".yaml") || strings.Contains(data.policyConfiguration, ".yml") || strings.Contains(data.policyConfiguration, ".json") {
+				policyBytes, err := os.ReadFile(data.policyConfiguration)
```

Comment:
Go through afero so we can write a test for this?

Reply:
ack